.ci/jobs/beats-schedule-weekly.yml

```diff
@@ -19,7 +19,7 @@
             wipe-workspace: 'True'
             name: origin
             shallow-clone: true
-            credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+            credentials-id: github-app-beats-ci
```

Comment:
where does this change come from? :)

Reply:
https://github.com/elastic/beats/blob/main/.ci/jobs/beats.yml#L25 I took that from here 😬